ironic-deployment/ironic/ironic.yaml

```diff
@@ -56,6 +88,22 @@ spec:
           imagePullPolicy: Always
           command:
             - /bin/runironic-api
+          livenessProbe:
+           exec:
+             command: ["sh", "-c", "curl 127.0.0.1:6385 && curl 127.0.0.1:6180"]
```

Comment:
please check 6385/v1/ that should be enough. If Ironic API is answering, it means that httpd is running properly, no need to test 6180 I think

Reply:
Thanks, done.